lib/standard/string.nit

```diff
@@ -379,12 +379,29 @@ abstract class Text
 	end
 
 	# Trims trailing and preceding white spaces
-	# A whitespace is defined as any character which ascii value is less than or equal to 32
 	#
 	#     assert "  Hello  World !  ".trim   == "Hello  World !"
 	#     assert "\na\nb\tc\t".trim          == "a\nb\tc"
+	#
+	# `Char::is_whitespace` is used to determine what is a whitespace.
 	fun trim: SELFTYPE do return (self.l_trim).r_trim
 
+	# Is the string non-empty but only made of whitespaces?
+	#
+	#    assert " \n\t ".is_whitespace    == true
+	#    assert "  hello  ".is_whitespace == false
+	#    assert "".is_whitespace          == false
+	#
+	# `Char::is_whitespace` is used to determine what is a whitespace.
```

Comment:
This sentence is so passive, I had to use a monocle to read it!

~~~nit
# Uses `Char::is_whitespace` to determine what is a whitespace.`
~~~